framework/src/preference/shared_preference.c

```diff
@@ -198,6 +217,28 @@ int preference_shared_get_string(const char *key, char **value)
 	return ret;
 }
 
+int preference_shared_get_binary(const char *key, char **value)
```

Comment:
It would be good to provide size for the binary type. So that API user's can do the operations accordingly.

Reply:
Done